src/ipc/ipc4/helper.c

```diff
@@ -949,6 +949,8 @@ __cold static const struct comp_driver *ipc4_get_drv(const void *uuid)
 	struct comp_driver_info *info;
 	uint32_t flags;
 
+	(void)sof_uuid; /* Avoid possible warning of unused */
```

Comment:
@singalsu  You should be able to use "__maybe_unused" attribute for these cases.  See examples in Zephyr and SOF, often used for variables that are checked in asserts and log statements that may be omitted in production builds.

Reply:
Ah that was it, I had vague memory of something like that but I could not find example. I'll change, stay tuned.

Reply:
Updated now, it looks like the stubs build passed with green check mark with `__maybe_unused`.